tokio-util/src/io/sync_bridge.rs

```diff
@@ -0,0 +1,99 @@
+use std::io::{Read, Write};
+use tokio::io::{AsyncRead, AsyncReadExt, AsyncWrite, AsyncWriteExt};
+
+/// Use a [`tokio::io::AsyncRead`] synchronously as a [`std::io::Read`] or
+/// a [`tokio::io::AsyncWrite`] as a [`std::io::Write`].
+#[derive(Debug)]
+pub struct SyncIoBridge<T> {
```

Comment:
Would probably be worth to document that you can use it with non-Unpin types by doing `SyncIoBridge::new(Box::pin(io))`

Reply:
Yep, done!